about.html

```diff
@@ -0,0 +1,123 @@
+                    <li class="nav-item"><a href="about.html" class="nav-link ">About</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Sponser</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                </ul>
+    
+
+        <section class="about-headline d-flex flex-column gap-3 justify-content-between  align-items-center text-center  p-5 m-auto">
+            <div><h3 class=" h-grd">CREATIVE COMNMONS<br>GLOBAL EVENT 2023</h3></div>
+            <p class="about-para p-font border border-3 border-light p-3 bg-white">This Event will show us that the next big thing isn’t a new technology, it’s a new idea.
+                 Human Security and technology<br> in all lief aspect is that idea. The Human Security concept also includes access to healthcare, <br>economic security, environmental 
+                 protection including resiliency and sustainability, personal safety and mobility, community security,<br> access to education,
+                  broadband access and political freedom</p>
+                <span class=" text-muted"> Please contact us per Email for nay further questions about CC Global Summit 2015! </span>
+                  <p class="mt-2  text-muted" > techno2023@cckorea.org</p>
+        </section>
+        </div>
+
+        <section class="container-fluid p-5 bg-light logo">
+            <div class="d-flex flex-column flex-wrap gap-5 align-items-center">
+                <h3 class="text-center">Creative Commons Global Event 2023 Logo</h3>
+                <p class="text-center">The logo of TechNo Global Event 2023 was decided through the logo competition from 8, June to 7, July.<br>
+                    After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.<br>
+                    After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s event as follows:</p>
+                    <img src="imgs/logo.jpg" width="300px" alt="logo img"/>
+            </div>
+        </section>
+         
+        <section class=" container-fluid bg-white">
+            <div class="d-flex flex-column gap-2 p-5 m-auto justify-content-around align-items-center">
+            <h3 class="text-center">See the past CC Global Summits</h3>
+            <p class="text-center">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+            <div class="d-sm-flex flex-lg-row flex-md-row flex-sm-column align-items-center justify-content-between gap-3 about-past">
+                <img src="imgs/paris.png" alt="first event " width="432px" height="236px">
+                <img src="imgs/morocco.png" alt="second event" width="432px" height="236px">
+            </div>
```

Comment:
- [ ] Kindly make your images here have small width since it has a bigger width in your design but it has a smaller in the given design so kindly stick with the design requirements as much as possible 🔥💯🤩

- [ ] I highly recommend to add the date on the images as shown in the given design so you can add the date of the conference like 2018 or 2019 💯🤩
- Yours
![image](https://user-images.githubusercontent.com/58553711/227085893-5895ec50-264f-45d8-a7c2-6f29d2876210.png)


- Given

![image](https://user-images.githubusercontent.com/58553711/227085937-5f7f729d-358b-49d1-a5b5-722d25bb00a9.png)